django_freeradius/tests/test_api.py

```diff
@@ -52,3 +53,140 @@ def test_postauth_400(self):
+                                    {'username': 'admin', 'NAS-IP-Address': '172.16.64.91',
+                                     'NAS-Port': "1",  'Called-Station-Id': '00-27-22-F3-FA-F1:hostname',
+                                     'Calling-Station-Id': '5c:7d:c1:72:a7:3b',  'NAS-Identifier': '',
+                                     'Acct-Status-Type': 'Start', 'Acct-Authentic': 'RADIUS',
+                                     'Acct-Delay-Time': '0',
+                                     'Acct-Unique-Session-Id': '75058e505ac30c5b4b9c686ee853b70f',
+                                     'Acct-Terminate-Cause': 'User-Request',  'Acct-Input-Octets': '9900909',
+                                     'Acct-Output-Octets': '1511075509',  'NAS-Port-Type': 'Async',
+                                     'Acct-Session-Time': '261', 'Login-Service': 'Telnet',
+                                     'Login-IP-Host': '172.16.64.25', 'Acct-Session-Id': '35000006',
+                                     'Framed-Protocol': '', 'Framed-IP-Address': '',
+                                     'Service-Type': 'Login-User', 'Realm': '',
+                                     'Acct-Authentic': 'RADIUS'})
+        self.assertEqual(RadiusAccounting.objects.filter(session_id='35000006').count(), 1)
+        self.assertEqual(RadiusAccounting.objects.all().count(), 1)
+        response = self.client.post(reverse('freeradius:accounting'),
+                                    {'username': 'admin', 'NAS-IP-Address': '172.16.64.91',
+                                     'NAS-Port': "2",  'Called-Station-Id': '00-27-22-F3-FA-F1:hostname',
+                                     'Calling-Station-Id': '5c:7d:c1:72:a7:3b',  'NAS-Identifier': '',
+                                     'Acct-Status-Type': 'Interim-Update', 'Acct-Authentic': 'RADIUS',
+                                     'Acct-Delay-Time': '0',
+                                     'Acct-Unique-Session-Id': '75058e505ac30c5b4b9c686ee853b70f',
+                                     'Acct-Terminate-Cause': 'User-Request',  'Acct-Input-Octets': '9810909',
+                                     'Acct-Output-Octets': '1622073403',  'NAS-Port-Type': 'Async',
+                                     'Acct-Session-Time': '263', 'Login-Service': 'Telnet',
+                                     'Login-IP-Host': '172.16.64.25', 'Acct-Session-Id': '35000006',
+                                     'Framed-Protocol': '', 'Framed-IP-Address': '',
+                                     'Service-Type': 'Login-User', 'Realm': '',
+                                     'Acct-Authentic': 'RADIUS'})
+        self.assertEqual(RadiusAccounting.objects.filter(session_id='35000006').count(), 1)
+        self.assertEqual(RadiusAccounting.objects.all().count(), 1)
+        self.assertEqual(response.status_code, 204)
+        self.assertEqual(response.data, {''})
+
+    def test_accounting_stop(self):
+        RadiusAccounting.objects.all().count()
```

Comment:
I think you should remove this line because is redundant with the test_accounting_start.

As a general rule redundant tests are not very useful, testing more use cases is useful. Concise and easy to read tests are useful.